packages/core/lib/combo/settings/data.ts

```diff
@@ -881,6 +881,13 @@ export const SETTINGS = [{
   type: 'boolean',
   description: 'Makes the Mask Shop in Market open during the night',
   default: false
+}, {
+  key: 'crossAge',
+  name: 'Cross-Games Age',
+  category: 'main.cross',
+  type: 'boolean',
+  description: 'When you enter MM as Adult Link, you will be Adult Link in MM. Model credit: Skilar.',
```

Comment:
Credits shouldn't be here, or else people are gonna start adding credits for every asset/texture/logic/whatnot and it'll get unreadable (see old BSD software where the credits is larger than the documentation).

The model file should go in its own commit, commited by you but authored by Skilar, or mentionned in the commit descr, or both.

Once we add in-game credits we can list him there too.

Reply:
Ok I will amend the commits.